gammapy/background/models.py

```diff
@@ -71,3 +82,740 @@ def evaluate(self, x, y):
+        a_dety_hi = Quantity([self.dety_bins[1:]])
+        a_energy_lo = Quantity([self.energy_bins[:-1]])
+        a_energy_hi = Quantity([self.energy_bins[1:]])
+        a_bg = Quantity([self.background])
+
+        # table
+        table = Table()
+        table['DETX_LO'] = a_detx_lo
+        table['DETX_HI'] = a_detx_hi
+        table['DETY_LO'] = a_dety_lo
+        table['DETY_HI'] = a_dety_hi
+        table['ENERG_LO'] = a_energy_lo
+        table['ENERG_HI'] = a_energy_hi
+        table['Bgd'] = a_bg
+
+        table.meta['E_THRES'] = a_energy_lo.flatten()[0].value
+
+        table.meta['name'] = 'BACKGROUND'
+
+        return table
+
+    def to_fits_bin_table(self):
+        """Convert cube background model to binary table fits format.
+
+        Returns
+        -------
+        tbhdu : `~astropy.io.fits.BinTableHDU`
+            table containing the bg cube
+        """
+        return _table_to_fits_bin_table(self.to_table())
+
+    def to_fits_image(self):
+        """Convert cube background model to image fits format.
+
+        Returns
+        -------
+        hdu_list : `~astropy.io.fits.HDUList`
+            HDU list with:
+
+            * one `~astropy.io.fits.PrimaryHDU` image for the bg cube
+
+            * one `~astropy.io.fits.BinTableHDU` table for the energy binning
+        """
+        # data
+        imhdu = fits.PrimaryHDU(data=self.background.value,
+                                header=self.det_wcs.to_header())
+        # add some important header information
+        imhdu.header['BG_UNIT'] = '{0.unit:FITS}'.format(self.background)
+        imhdu.header['CTYPE3'] = 'ENERGY'
+        imhdu.header['CUNIT3'] = '{0.unit:FITS}'.format(self.energy_bins)
+        imhdu.header['E_THRES'] = self.energy_bins[0].value
+
+        # get WCS object and write it out as a FITS header
+        wcs_header = self.det_wcs.to_header()
+
+        # get energy values as a table HDU, via an astropy table
+        energy_table = Table()
+        energy_table['ENERGY_BIN_EDGES'] = self.energy_bins
+        energy_table.meta['name'] = 'ENERGY_BINS'
+
+        enhdu = _table_to_fits_bin_table(energy_table)
+
+        hdu_list = fits.HDUList([imhdu, enhdu])
+
+        return hdu_list
+
+    def write(self, outfile, format='table', **kwargs):
+        """Write cube background model to fits file.
+
+        Several output formats are accepted, depending on the value
+        of the `format` parameter:
+
+        * bin_table (default and preferred format): `~astropy.io.fits.BinTableHDU`
+        * image (alternative format): `~astropy.io.fits.PrimaryHDU`,
+          with the energy binning stored as `~astropy.io.fits.BinTableHDU`
+
+        Depending on the value of the `format` parameter, this
+        method calls either `~astropy.io.fits.BinTableHDU.writeto` or
+        `~astropy.io.fits.HDUList.writeto`, forwarding the
+        `kwargs` arguments.
+
+        Parameters
+        ----------
+        outfile : `~str`
```

Comment:
OK, fixed. Though I think it could be convenient sometimes in order to quickly check for a list of methods, and I don't think it harms to have these links.